### ISSUE TITLE
Some users seeing missing text on tech hover (since Sept. builds)

### ISSUE THREAD

**Issue by alleryn** [freeorion/freeorion#1878]
### Environment
* **FreeOrion Version**: 8ee7fb7 (Sept 4) - f5304af (Nov 24)
* **Operating System**: Windows 10 Home
* **Graphic card used**: Nvidia Geforce GTX 1050
* **Fetched as**
  * Weekly development build (8ee7fb7)
  * Compiled from source (f5304af)

### Description
See also the forum thread http://www.freeorion.org/forum/viewtopic.php?f=28&t=10794.

In the Research screen, upon hovering over a tech, a box appears with the tech's icon on the left.  The information on the right side of the box is missing for some users.  (Possibly affecting Windows but not Linux users).

The problem did not occur in the official 0.4.7.1 release (139ffd9), or in the most recent weekly development build (4e12b03 Aug. 24) pre-dating the latest set of release candidates.  It does occur in 8ee7fb7 (Sept. 4) and going forward.

Working:
![fotechhover2](https://user-images.githubusercontent.com/30762579/33236073-0da34504-d20e-11e7-95fe-03b7a7fb34cf.jpg)
[freeorion.log](https://github.com/freeorion/freeorion/files/1503521/freeorion.log)
[freeoriond.log](https://github.com/freeorion/freeorion/files/1503522/freeoriond.log)

Not working:
![fotechhover1](https://user-images.githubusercontent.com/30762579/33236092-c51fd2e2-d20e-11e7-8fd9-2994c4913035.jpg)
[freeorion.log](https://github.com/freeorion/freeorion/files/1503530/freeorion.log)
[freeoriond.log](https://github.com/freeorion/freeorion/files/1503531/freeoriond.log)

### Expected Result
Text should appear.

### Steps to reproduce
* Open research screen.
* Hover your mouse over a tech.
* Wait some fractions of a second for the box to pop open.
* Issue occurs.

**Comment by dbenage-cx** [comment]
@alleryn Ty for report

Any difference with:
```diff
diff --git a/UI/TechTreeWnd.cpp b/UI/TechTreeWnd.cpp
index 74b021eb1..3c52a1593 100644
--- a/UI/TechTreeWnd.cpp
+++ b/UI/TechTreeWnd.cpp
@@ -133,7 +133,7 @@ std::shared_ptr<GG::BrowseInfoWnd> TechRowBrowseWnd(const std::string& tech_name
 
     std::string main_text;
 
-    main_text += UserString(tech->Category()) + " \u2013 ";  // u2013 = 'En dash'
+    main_text += UserString(tech->Category()) + " - ";
     main_text += UserString(tech->ShortDescription()) + "\n";
 
     if (empire) {
```

If not, does switching to tree view have any effect?

**Comment by alleryn** [comment]
The problem still occurs in tree view.  Working on the other.

**Comment by geoffthemedio** [comment]
Replacing `" \u2013 "` with `" "` fixed it for me.

**Comment by dbenage-cx** [comment]
Fixed in cbf1e48
